src/pages/mobile/VaultSetupThisDeviceSync.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <MobilePage
+    class="vault-setup-this-device-sync"
+    hide-tab-bar
+    left-button-icon-name="back"
+    right-button-icon-name="close"
+    :left-button-to="{ name: 'vault-setup-this-device' }"
+    :right-button-to="{ name: 'apps' }"
+  >
+    <template slot="title">
+      AirGap Setup
+      <AeFraction
+        numerator="3"
+        denominator="3"
+      />
+    </template>
+    <Guide fill="alternative">
+      <AeFraction
+        slot="icon"
+        numerator="3"
+        denominator="3"
+      />
+      <em>Vault sync on the way</em>
+      <br>Paste the sync code below.
+    </Guide>
+
+    <form
+      :id="_uid"
+      @submit.prevent="parseSyncCode"
+    >
+      <AeInputSync
+        v-model="syncCode"
+        v-validate="'required'"
```

Comment:
make a custom validator for sync code (use the same code with sync code parsing)